server/app.js

```diff
@@ -33,14 +33,14 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(bodyParser.urlencoded({ extended: false }));
-app.use(express.static(path.join(__dirname, 'public')));
+app.use(express.static(path.join(__dirname, 'build')));
 
 /* 
  * If testDB is true, uses a locally hosted mongoDB
  * Otherwise, it will use a cloud hosted DB set in the .env file
  * MongoDB must be installed
  */
-const testDB = true;
+const testDB = process.env.DEV_MODE;
```

Comment:
To prevent truthy string weirdness, should probably be testDB = (process.env.DEV_MODE.toLowerCase() === “true”);